include/dxc/DXIL/DxilConstants.h

```diff
@@ -1888,7 +1888,10 @@ const uint64_t ShaderFeatureInfo_WriteableMSAATextures = 0x40000000;
 // WaveMMA slots in between two SM 6.6 feature bits.
 const uint64_t ShaderFeatureInfo_WaveMMA = 0x8000000;
 
-const unsigned ShaderFeatureInfoCount = 31;
+const uint64_t ShaderFeatureInfo_SampleCmpGradientOrBias = 0x80000000;
+const uint64_t ShaderFeatureInfo_ExtendedCommandInfo = 0x100000000;
```

Comment:
For the record, I don't think the name of this feature bit is intuitive at all.

Reply:
Agreed. It's a remnant from what it was before.